tasks/vhost-www.yml

```diff
@@ -2,7 +2,7 @@
     name: httpd
     tasks_from: pki-tls
   vars:
-    pki_hostname: "{{ httpd_www_hostname }}"
+    pki_hostname: www.centos.org
```

Comment:
Let's keep that a variable, as it can be different for dev/stg and prod ENVs, so nothing hard-coded